Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs

```diff
@@ -33,6 +42,47 @@ private void InitializeInjector()
         SubscribeLocalEvent<InjectionCancelledEvent>(OnInjectionCancelled);
     }
 
+    private void AddSetTransferVerbs(EntityUid uid, InjectorComponent component, GetVerbsEvent<AlternativeVerb> args)
+        {
+            if (!args.CanAccess || !args.CanInteract || args.Hands == null)
+                return;
+
+            if (!EntityManager.TryGetComponent<ActorComponent?>(args.User, out var actor))
+                return;
+
+            // Custom transfer verb
+            AlternativeVerb custom = new();
+            custom.Text = Loc.GetString("comp-solution-transfer-verb-custom-amount");
+            custom.Category = VerbCategory.SetTransferAmount;
+            custom.Act = () => component.UserInterface?.Open(actor.PlayerSession);
+            custom.Priority = 1;
+            args.Verbs.Add(custom);
```

Comment:
In SS13, it only allows 5, 10, and 15 for syringes. No custom amounts. For smaller amounts, players would use a dropper for 1-5. Should that be replicated here or should it just all be done using syringes?

Reply:
Yeah I'll strip out the custom amount.